### DISCUSSION TITLE
Is there a way to hide the console for Windows Skia App

### DISCUSSION THREAD

**Discussion by sosuisen** [unoplatform/uno#17221] [Q&A] [answered]
When I launch a net8.0-desktop application, the console is displayed along with the application. Displaying the console is not very common behavior for Windows applications. Is there a way to either not display the console or launch the application with the console minimized?

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the question. The console is removed when running in release mode, if you want to remove in debug, you'll need to set the `<OutputType>` to `WinExe` only for `net8.0-desktop`.

**Reply by sosuisen** [reply]
Release mode works well. Thank you.
